_includes/releases/v22.1/v22.1.0-beta.5.md

```diff
@@ -0,0 +1,51 @@
+## v22.1.0-beta.5
+
+Release Date: May 3, 2022
+
+{% include releases/release-downloads-docker-image.md release=include.release %}
+
+<h3 id="v22-1-0-beta-5-security-updates">Security updates</h3>
+
+- `Crdb_internal.reset_sql_stats()` and `crdb_internal.reset_index_usage_stats()` builtins now check if user has the admin role. [#80384][#80384]
+- [#80248][#80248] [7d55af0e6][7d55af0e6]  (Raphael 'kena' Poss <knz@cockroachlabs.com>)
+
+<h3 id="v22-1-0-beta-5-enterprise-edition-changes">Enterprise edition changes</h3>
+
+- Backups run by secondary tenants now write protected timestamp records to protect their target schema objects from garbage collection during backup execution. [#80670][#80670]
+
+<h3 id="v22-1-0-beta-5-sql-language-changes">SQL language changes</h3>
+
+- The cluster settings `cloudstorage.<provider>.read.node_rate_limit` and `cloudstorage.<provider>.read.node_burst_limit` can now be used to limit throughput when reading from cloud storage during a `RESTORE` or `IMPORT`. [#80281][#80281]
+
+<h3 id="v22-1-0-beta-5-bug-fixes">Bug fixes</h3>
+
+- Addressed issue where automatic encryption-at-rest data key rotation would get disabled after a node restart without a store key rotation. [#80564][#80564]
+
+- Fixed a bug whereby the cluster version could regress due to a race condition. [#80712][#80712]
+
+<h3 id="v22-1-0-beta-5-performance-improvements">Performance improvements</h3>
+
+- Bulk ingestion of unsorted data during `IMPORT` and schema changes now uses a higher level of parallelism to send produced data to the storage layer. [#80487][#80487]
+
+<h3 id="v22-1-0-beta-5-miscellaneous">Miscellaneous</h3>
+
+<h4 id="v22-1-0-beta-5-docker">Docker</h4>
+
+- Refactored the initialization process of the docker image to accomodate the use case with memory storage. [#80558][#80558]
```

Comment:
I'm not sure what this "the use case with memory storage" means here? Maybe rework this note to align with what the change is and why user's need to know.

"docker" should be capped